apps/site/load_tests/locustfile.py

```diff
@@ -1,6 +1,5 @@
-from locust import HttpLocust, TaskSet, task
-from page import Page
```

Comment:
It looks like the `page.py` file is no longer used anywhere and can probably just be deleted.